src/main/resources/templates/plan/edit.html

```diff
@@ -1,10 +1,147 @@
+               class="rounded-t-md text-center items-center cursor-pointer">
+              <span th:text="${editDayForm.day} + '日目'"></span>
+            </a>
+          </div>
+        </div>
+        <div class="flex flex-col mt-6 mb-10">
+          <div class="mb-6">
+            <p class="mb-2 sm:text-lg text-base">出発地点<span class="sm:ml-2 ml-1 text-sm text-danger">必須</span></p>
+            <!-- TODO:出発地点を表示する用modalを挿入 -->
+            <th:block th:replace="~{fragment/modal :: start}"/>
+          </div>
+          <div class="mb-6">
+            <p class="mb-2 sm:text-lg text-base">目的地</p>
+            <!-- TODO:目的地の表示をするmodalを挿入する -->
+            <div th:each="place : ${places}">
+              <th:block th:replace="~{fragment/modal :: place}"/>
+            </div>
+          </div>
+          <div class="mb-6">
+            <p class="mb-2 sm:text-lg text-base">終了地点<span class="sm:ml-2 ml-1 text-sm text-danger">必須</span></p>
+            <!-- TODO:終了地点を表示するmodalを挿入 -->
+            <th:block th:replace="~{fragment/modal :: start}"/>
+          </div>
+          <!-- 目的地を追加するためのモーダル -->
+          <th:block th:replace="~{fragment/modal :: places(1)}"/>
+          <!-- おすすめ目的地のセクション -->
+          <section class="pt-10">
+            <details open class="p-4 group overflow-hidden transition-[max-height] duration-500">
+              <summary class="sm:text-xl text-lg cursor-pointer font-semibold marker:text-transparent group-open:before:rotate-90  before:origin-center relative before:w-[18px] before:h-[18px] before:transition-transform before:duration-200 before:-left-1 before:top-2/4 before:-translate-y-2/4 before:absolute before:bg-no-repeat before:bg-[length:18px_18px] before:bg-center before:bg-[url('data:image/svg+xml,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20class%3D%22h-6%20w-6%22%20fill%3D%22none%22%20viewBox%3D%220%200%2024%2024%22%20stroke%3D%22currentColor%22%20stroke-width%3D%222%22%3E%0A%20%20%3Cpath%20stroke-linecap%3D%22round%22%20stroke-linejoin%3D%22round%22%20d%3D%22M9%205l7%207-7%207%22%20%2F%3E%0A%3C%2Fsvg%3E')]">
+                おすすめ目的地を追加
+              </summary>
+              <hr class="my-2 scale-x-150"/>
+              <div class="sm:text-lg text-base -m-4 -mt-2 p-4">
+                目的地のジャンル別に絞る
+                <!-- TODO:正規のジャンル一覧はControllerから受け渡す -->
+                <select class="focus:outline-none ml-6 rounded-xl h-8 text-xs" name="sample">
+                  <option value="0">全て</option>
+                  <option value="1">観光</option>
+                  <option value="2">グルメ</option>
+                  <option value="3">ショッピング</option>
+                  <option value="4">アクティビティ</option>
+                  <option value="5">宿泊</option>
+                  <option value="6">その他</option>
+                </select>
```

Comment:
```suggestion
 <select class="focus:outline-none ml-6 rounded-xl h-8 text-xs" name="genre">
                                    <option value="">ジャンルを選択</option>
                                    <th:block th:each="genre : ${genres}">
                                        <option th:value="${genre.id}" th:text="${genre.name}"></option>
                                    </th:block>
                                </select>
```
ここもcreate.htmlと同じ感じでよさそう